src/DynamoCore/DynamoCore.csproj

```diff
@@ -365,14 +365,14 @@ limitations under the License.
   <Target Name="BeforeBuild">
     <ItemGroup>
       <ExternProtoGeometry Include="$(SolutionDir)..\extern\ProtoGeometry\*" />
-      <ExternProtoGeometryXml Include="$(SolutionDir)..\extern\ProtoGeometry\ProtoGeometry.XML" />
+      <LibGInterfaceDll Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_225_0_0.2.1.0\ProtoGeometry\LibG.Interface.dll" />
+      <ProtoGeometryXml Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_225_0_0.2.1.0\lib\ProtoGeometry\ProtoGeometry.XML" />
       <ExternProtoGeometryUICulture Include="$(SolutionDir)..\extern\ProtoGeometry\$(UICulture)\*" />
       <ExternProtoGeometryLibGLocale Include="$(SolutionDir)..\extern\ProtoGeometry\libg_locale\**\*.*" />
-      <ExternLibG223 Include="$(SolutionDir)..\extern\LibG_223_0_1\*" />
-      <ExternLibG224 Include="$(SolutionDir)..\extern\LibG_224_0_1\*" />
-      <ExternLibG2244 Include="$(SolutionDir)..\extern\LibG_224_4_0\*" />
-      <ExternLibG225 Include="$(SolutionDir)..\extern\LibG_225_0_0\*" />
-      <ExternLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
+      <LibG223 Include="$(SolutionDir)\packages\DynamoVisualProgramming.LibG_223_0_1.2.1.0\LibG_223_0_1\*" />
```

Comment:
I worry that each time we bump up the version numbers of LibG builds, these package names would update as well and these entries would have to be manually updated over here. I'll try and see if there's a way we can parameterize this.